spec.bs

```diff
@@ -787,7 +810,7 @@ An <dfn>navigation API method navigation</dfn> is a [=struct=] with the followin
 
   1. Let |info| be |options|["{{NavigationOptions/info}}"] if it exists; otherwise, undefined.
 
-  1. Let |historyHandling| be "<a for="history handling behavior">`replace`</a>" if |options|["{{NavigationNavigateOptions/replace}}"] is true; otherwise, "<a for="history handling behavior">`default`</a>".
+  1. Let |historyHandling| be "<a for="history handling behavior">`replace`</a>" if |options|["{{NavigationNavigateOptions/history}}"] is "{{NavigationHistoryBehavior/replace}}" or if |document| is not <a spec="HTML">completely loaded</a>; otherwise, "<a for="history handling behavior">`default`</a>".
```

Comment:
This is somewhat of a bugfix, in the sense that Chromium definitely already did this conversion, and I believe we have tests for it.

In Chromium, by default all before-load navigations are replacements, with a special exception for anchor clicks: see https://source.chromium.org/chromium/chromium/src/+/main:third_party/blink/renderer/core/frame/local_frame.cc;l=454;drc=a24520a700e2581395920c74d59a918aa60bd5c9 . However in the spec, each call site of the navigate algorithm independently changes historyHandling, except anchor clicks which do not.